l10n_ch_base_bank/README.rst

```diff
@@ -84,16 +84,23 @@ Authors
 Contributors
 ~~~~~~~~~~~~
 
-* Nicolas Bessi (Camptocamp)
-* Vincent Renaville <vincent.renaville@camptocamp.com>
-* Joël Grand-Guillaume <joel.grandguillaume@camptocamp.com>
-* Guewen Baconnier <guewen.baconnier@camptocamp.com>
 * Paul Catinean <paulcatinean@gmail.com>
-* Yannick Vaucher <yannick.vaucher@camptocamp.com>
-* Akim Juillerat <akim.juillerat@camptocamp.com>
-* Iryna Vyshnevska <i.vyshnevska@mobilunity.com>
 * Simone Vanin <simone.vanin@agilebg.com>
 
+* `Camptocamp <https://camptocamp.com>`_:
+
+    * Nicolas Bessi (Camptocamp)
+    * Vincent Renaville <vincent.renaville@camptocamp.com>
+    * Joël Grand-Guillaume <joel.grandguillaume@camptocamp.com>
+    * Guewen Baconnier <guewen.baconnier@camptocamp.com>
+    * Yannick Vaucher <yannick.vaucher@camptocamp.com>
+    * Akim Juillerat <akim.juillerat@camptocamp.com>
+    * Iryna Vyshnevska <iryna.vyshnevska@camptocamp.com>
+
+* `Trobz <https://trobz.com>`_:
+
+    * Son Ho <sonhd@trobz.com>
```

Comment:
Please don't change the structure of contributors, the list was ordered by date of contribution, grouping by company breaks that order.

Reply:
Hi, I just updated it, please check again